avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastDeserializerGenerator.java

```diff
@@ -144,6 +146,9 @@
     } catch (JClassAlreadyExistsException e) {
       throw new FastDeserializerGeneratorException("Class: " + className + " already exists");
     } catch (Exception e) {
+      LOGGER.error("Failed to generate a deserializer.");
```

Comment:
better to have this as a single line (maybe with \n) because in a busy system other prints can get injected in the middle on the log file

Reply:
also - use {} and dont call toString() (less important for error, but still)

Reply:
We do not need to log here since the `FastDeserializerGeneratorException` would be catched in upper layer `buildGeneric/SpecificDeserializer` and schemas would be logged?

Reply:
Hmm... I think the thing is that we have test code that invokes this function directly, and therefore does not benefit from the upstream logging that happens in the main code. I will remove those extra logs then.